marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/map-multiple-entities/test-data/content/customerInfo.xml

```diff
@@ -0,0 +1,41 @@
+<Customer xmlns="customerInfo">
```

Comment:
Same comments above about the test data. Also, let's start adding new tests under /data-hub/5/mapping - e.g. data-hub/5/mapping/map-multiple-entities . The builtins/steps directory should contain a minimum of code that delegates to libraries under data-hub/5/mapping - likewise for the data-services directory. 

The mapping app code is fine for now, we can move that later. But let's start getting the tests under data-hub/5/mapping.